Wrapper.h

```diff
@@ -0,0 +1,59 @@
+#include "Animal.h"
+#include "FancyPrinter.h"
+#include "DividingStruct.h"
+
```

Comment:
- `#pragma once` is missing
- several types can be forward declared here, as they are not being used in this header file.
- the header file should be included in the cpp file
- [ ] revise

Reply:
I am having a problem with this, because I'm not clear on how you would like me to handle the inner UDTs (Clock and  MultiplierStruct). As far as I can tell, there isn't a way to forward declare these.